app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,24 +3,64 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assignee: null,
+  tags: [],
 };
 
-export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
-  title: yup.string().required("Title is required"),
-  description: yup.string().required("Description is required"),
-});
+export const ASSIGNEES = [
+  {
+    label: "User",
+    value: "user",
+  },
+  {
+    label: "Admin",
+    value: "admin",
+  },
+  {
+    label: "Developer",
+    value: "developer",
+  },
+];
 
-export const NOTES_TABLE_COLUMN_DATA = [
+export const TAGS = [
+  {
+    label: "Create",
+    value: "create",
+  },
+  {
+    label: "Read",
+    value: "read",
+  },
   {
-    title: "Title",
-    dataIndex: "title",
-    key: "title",
-    width: "30%",
+    label: "Update",
+    value: "update",
   },
   {
-    title: "Description",
-    dataIndex: "description",
-    key: "description",
-    width: "70%",
+    label: "Delete",
+    value: "delete",
```

Comment:
Go through the Slack thread. You should be building these options rather than hardcoding.